app/models/hyrax/virus_scanner.rb

```diff
@@ -30,12 +30,12 @@ def initialize(file)
     # Override this method to use your own virus checking software
     # @return [Boolean]
     def infected?
-      defined?(ClamAV) ? clam_av_scanner : null_scanner
+      defined?(Clamby) ? clam_av_scanner : null_scanner
```

Comment:
Changing the generators immediately seems good, but I think we need a transition plan for existing applications.

I guess technically we could sneak this into 3.0.0, but it seems late in the game for dropping support for the `ClamAV` gem.

What do you think about supporting both?